src/store/wallet/slice.ts

```diff
@@ -20,11 +20,13 @@ export type WalletAddress = string
 type WalletState = {
   balance: Nullable<StringWei>
   pendingClaimBalance: Nullable<StringWei>
+  localBalanceChange: boolean
 }
 
 const initialState: WalletState = {
   balance: null,
-  pendingClaimBalance: null
+  pendingClaimBalance: null,
+  localBalanceChange: false
```

Comment:
nit: wdyt about calling it something like `localBalanceDidChange`? When I first saw at it I assumed this was going to be a numeric value